### PR TITLE
Fix Skopeo tests with latest containers/image and kubernetes

### PR DESCRIPTION
**Pull request by erikh**
@runc0m here you go, this is based off erikh/image@kube-fix.

### CODE REVIEW

**Comment by runcom** [comment]
Let's see if tests pass here :) @mtrmac

File: vendor.conf [hunk 1]
```diff
@@ -27,6 +27,7 @@ github.com/mtrmac/gpgme master
 # openshift/origin' k8s dependencies as of OpenShift v1.1.5
 github.com/golang/glog 44145f04b68cf362d9c4df2182967c2275eaefed
 k8s.io/kubernetes 4a3f9c5b19c7ff804cbc1bf37a15c044ca5d2353 https://github.com/openshift/kubernetes
+k8s.io/apimachinery master
```
Comment (runcom) [inline vendor.conf]: @mtrmac do we need to use this from openshift?

**Comment by erikh** [comment]
success!

File: vendor.conf (hunk 1 above)
Comment (mtrmac) [inline vendor.conf]: Openshift doesn't have a forked repo of `k8s.io/apimachinery` right now, so we don’t really have a choice unless we wanted to force all developers of `containers/image` to maintain a `$GOPATH` with `k8s.io/kubernetes` checked out from the `openshift/kubernetes` fork.

Anyway, the …`pkg/util/errors` packages are identical between `k8s.io/apimachinery` and `master` of `openshift/kubernetes` (though we had an older fork in here). The …`pkg/util/net` packages differ a bit, but that seems trivial enough (HTTP 2 enabling/disabling mechanism).

**Comment by mtrmac** [comment]
LGTM.

After https://github.com/containers/image/pull/214 is merged, please restore the `github.com/containers/image master` line, rerun `vndr` and update this PR.

**Comment by runcom** [comment]
@erikh please go ahead and restore `github.com/containers/image master` line, re-run `vndr` an update this PR (as @mtrmac said). Thanks!

**Comment by mtrmac** [comment]
Merged in #291. Thanks for the fix!

**Pull request closed without merging**